config/mempool.mk

```diff
@@ -21,7 +21,17 @@ num_cores_per_tile ?= 4
 banking_factor ?= 4
 
 # Radix for hierarchical AXI interconnect
-axi_hier_radix ?= 20
+axi_hier_radix ?= 17
```

Comment:
I checked, and this change should not impact the backend too much, so if it actually helps with the DRAM's performance, we can keep it that way. But it would be interesting to know why we can't get full performance with smaller bursts on the DRAM.